### PR TITLE
Recently published files API --> api docs

### PR DESCRIPTION
**Pull request by anosulchik**

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -10568,6 +10621,35 @@ components:
...
+      properties:
+        fileUri:
+          description: The URI of the file that was published
+          type: string
+        publishDate:
+          description: The date the file was published for the specified locale
+          format: 'YYYY-MM-DDThh:mm:ssZ'
+          type: string
+        localeId:
+          description: A unique identifier for the published file
```
Comment (dnetrebenko-smartling) [inline spec/openapi.yaml:10646]: wrong description

**Review by dnetrebenko-smartling — CHANGES_REQUESTED** [review]

File: spec/openapi.yaml (hunk 1 above)
Comment (anosulchik) [reply spec/openapi.yaml]: Right, thank you.

**Comment by anosulchik** [comment]
@dnetrebenko-smartling, please check latest changes and unblock PR if all looks good.

File: spec/openapi.yaml [hunk 2]
```diff
@@ -2257,6 +2257,66 @@ paths:
...
+          type: string
+      - description:
+          Limits the response to only those files that have a ```publishDate```
+          date after the ```publishedAfter``` parameter. The items array can
+          be empty if no files match this criteria.
+        in: query
+        name: publishedAfter
+        required: true
+        schema:
+          format: 'YYYY-MM-DDThh:mm:ssZ'
```
Comment (dnetrebenko-smartling) [inline spec/openapi.yaml:2285]: I don't see a description for `fileUri` parameter

File: spec/openapi.yaml [hunk 3]
```diff
@@ -2257,6 +2257,66 @@ paths:
...
+          $ref: '#/components/responses/Error401ResponseDefinition'
+        429:
+          $ref: '#/components/responses/Error429ResponseDefinition'
+        500:
+          $ref: '#/components/responses/Error500ResponseDefinition'
+      x-code-samples:
+      - lang: curl
+        source: >
+          curl -H "Authorization: Bearer $smartlingToken" -F
+          "file=@filename.properties" -F 'fileUri=filename.properties'
```
Comment (dnetrebenko-smartling) [inline spec/openapi.yaml:2317]: what does `"file=@filename.properties"` mean here? IMO, it's just a copy-paste...

File: spec/openapi.yaml (hunk 2 above)
Comment (anosulchik) [reply spec/openapi.yaml:2285]: Hmmm, this API accepts inly publishedAfter param and projectId (path param), why do you think we should describe fileUri here?
https://github.com/Smartling/published-files-service/blob/master/routes/recentlyPublishedFiles.js#L46

File: spec/openapi.yaml (hunk 3 above)
Comment (anosulchik) [reply spec/openapi.yaml]: Holly cow, this is a good catch. Fixed.

File: spec/openapi.yaml (hunk 2 above)
Comment (dnetrebenko-smartling) [reply spec/openapi.yaml:2285]: never mind... my mistake

**Review by dnetrebenko-smartling — APPROVED** [review]

**Pull request merged**